build/Dockerfile

```diff
@@ -20,9 +20,9 @@ RUN chmod +x /entrypoint.sh
 RUN pipx install .
 
 RUN mkdir -p /config
-COPY build/config_remote.yaml /config/config_remote.yaml
+COPY config.yaml /config/config.yaml
```

Comment:
Agree with the need to reduce the number of config files, however, the config_remote.yaml is a unique yaml in that it points to a CDF extraction pipeline that in turn contains its own configuration file that is subsequently loaded after the remote is processed. See [here](https://carboncapture.fusion.cognite.com/carbon-platform-sandbox/extpipes/extpipe/7217?cluster=westeurope-1.cognitedata.com&env=westeurope-1) for example. The result is that you can manipulate the configuration directly from CDF without touching the image or even the environment variables.

I'm not sure as to the best way to proceed, ideally it would be easy to support either the remote config or the full config in the image. But I feel like just removing the remote config eliminates this handy functionality that is built into CDF.

Reply:
I agree, we need to cut down on the number of configs, but i dont want to remove functionality